e2e/utils/fixture.ts

```diff
@@ -1,17 +1,18 @@
 import { joinPathFragments, readJsonFile, writeJsonFile } from "@nrwl/devkit";
 import { exec, spawn } from "child_process";
-import { ensureDir, ensureDirSync, readFile, remove, writeFile } from "fs-extra";
+import { ensureDir, ensureDirSync, existsSync, readFile, remove, writeFile } from "fs-extra";
 import isCI from "is-ci";
+import { dump } from "js-yaml";
 import { dirSync } from "tmp";
 
-interface RunCommandOptions {
+export interface RunCommandOptions {
```

Comment:
How come this was exported as part of the changes, I don't think it is used outside this file?

Reply:
`RunCommandOptions` and `RunCommandResult` are part of the public interface of `Fixture` because they are parameters / return values of Fixture's functions.

Reply:
Removed the exports